app/models/game.rb

```diff
@@ -1,8 +1,22 @@
 class Game < ActiveRecord::Base
+  has_many :reviews
+  belongs_to :user
+
+  COMPLEXITIES = {
+    1 => "low",
+    2 => "medium",
+    3 => "high",
+    4 => "extreme"
+  }
 
   validates :name, presence: true, uniqueness: true
+  validates :user_id, presence: true
```

Comment:
You can validate the user rather than the user ID. This ensures it is an actual user record rather than some arbitrary integer:

```ruby
validates :user, presence: true
```